src/js/shield_defs.js

```diff
@@ -2397,6 +2397,14 @@ export function loadShields(shieldImages) {
     (county) => (shields[`US:WY:${county}`] = pentagonShieldBlueYellow)
   );
 
+  // AFRICA
```

Comment:
Ah, I thought we were trying to put the continents in alphabetical order but sorting “North America” as “America, North”. Not a big deal either way, but this reminds me that this method and file need to be split apart at some point.